spec/requests/users_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+  describe "GET /users" do
+    it "returns http success" do
+      get "/users"
+      expect(response).to have_http_status(:success)
+      expect(@controller.view_assigns["users"]).not_to be_nil
```

Comment:
```suggestion
      expect(assigns(:users)).not_to be_nil
```

Além disso, esse seu teste ainda continua bem fraco. `be_nil` verifica se `@users` não é nil, porém o resultado é []. Eu sugiro criar um usuário no seu teste e verificar se ele é retornado na lista

https://relishapp.com/rspec/rspec-rails/docs/controller-specs